enterprise/app/repo/repo.css

```
@@ -1,16 +1,24 @@
 .create-repo-page {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans,
+    Droid Sans, Helvetica Neue, sans-serif;
```

not sure if you wanted this to match the main app font, but we're using Open Sans currently

```suggestion
  font-family: "Open Sans", sans-serif;
```

---

I'd like to try using this system-ui stack that we use for our marketing page. I messed around with making this the default for our whole app UI as well, but wanted to wait a bit and test it out before taking the leap.